src/zenml/integrations/__init__.py

```diff
@@ -44,6 +44,7 @@
 from zenml.integrations.sklearn import SklearnIntegration  # noqa
 from zenml.integrations.slack import SlackIntegration  # noqa
 from zenml.integrations.tensorflow import TensorflowIntegration  # noqa
+from zenml.integrations.vault import VaultSecretManagerIntegration  # noqa
```

Comment:
```suggestion
from zenml.integrations.vault import VaultSecretsManagerIntegration  # noqa
```

This should be plural: 'secrets manager' (and everywhere else it occurs)